manim/mobject/svg/code_mobject.py

```diff
@@ -336,9 +334,6 @@ def gen_colored_lines(self):
         code = Paragraph(
             *list(lines_text),
             line_spacing=self.line_spacing,
-            tab_width=self.tab_width,
-            font=self.font,
-            disable_ligatures=True,
```

Comment:
`font` and `disable_ligatures` seem to be *intended* arguments that forward to the internal `Text` that's used for `Paragraph`, but the issue is those arguments are also forwarded to `VGroup`. I'd probably suggest adding a `text_config` argument to `Paragraph` to properly separate these things, or maybe they could just not be forwarded to `VGroup`? Not sure, but I'd err on the side of the former.

Similarly thoughts for in `gen_line_numbers`